packages/compass-aggregations/src/components/saved-pipelines/saved-pipelines.tsx

```diff
@@ -82,7 +81,7 @@ function UnthemedSavedPipelines({
 }: SavedPipelinesProps) {
   return (
     <div className={savedPipelinesStyles}>
-      <Toolbar className={toolbarStyles}>
+      <div className={toolbarStyles}>
```

Comment:
Should this also be a `toolbar` in a `WorkspaceContainer` then? Same with QueryHistory.

Reply:
oh .. that would probably make sense, is the same layout right (something fixed on top and content that scrolls below)?

I thought we could unify this better with the way we intend to do modals though, and use `WorkspaceContainer` internally:

```
<InteractivePopover header="{stringOrElement}"></content>
```

Reply:
Sounds good, we can do it later in the project, definitely not a blocker here.